include/highfive/H5PropertyList.hpp

```diff
@@ -228,6 +228,7 @@ class Chunking {
 
   private:
     friend DataSetCreateProps;
+    friend GroupCreateProps;
```

Comment:
Why this?

https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetChunk

Says:
> hid_t plist | IN: Dataset creation property list                identifier.

And doesn't seem to talk about groups.

Reply:
So instruction L15 is invalid @matz-e ?
https://github.com/BlueBrain/HighFive/blob/b18aa78cc53818212e1f75332036f8fba9bc13bd/doc/poster/example2.cpp#L13-L16

In that case, it would be nice to have a trap that redirects the compiler to a `static_assert(cond, "This property doesn't apply to groups")`

Reply:
I think so, and I agree!

Reply:
Yeah that was my mistake initially, but fixed it later.